src/input_xml.F90

```diff
@@ -2489,6 +2494,102 @@ subroutine read_tallies_xml()
             ! Set to analog estimator
             t % estimator = ESTIMATOR_ANALOG
 
+          case ('mu')
+            ! Set type of filter
+            t % filters(j) % type = FILTER_MU
+
+            ! Set number of bins
+            t % filters(j) % n_bins = n_words - 1
+
+            ! Allocate and store bins
+            allocate(t % filters(j) % real_bins(n_words))
+            call get_node_array(node_filt, "bins", t % filters(j) % real_bins)
+
+            ! Allow a user to input a lone number which will mean that
+            ! you subivide [-1,1] evenly with the input being the number of bins
+            if (n_words == 1) then
+              Nangle = abs(int(t % filters(j) % real_bins(1)))
+              if (Nangle > 1) then
+                t % filters(j) % n_bins = Nangle
+                dangle = TWO / (real(Nangle,8))
+                deallocate(t % filters(j) % real_bins)
+                allocate(t % filters(j) % real_bins(Nangle + 1))
+                do iangle = 1, Nangle + 1
```

Comment:
I would change the upper index to Nangle since you explicitly set Nangle + 1 afterwards